src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -184,9 +184,8 @@ public partial struct JsonReaderState
     }
     public static partial class JsonSerializer
     {
-#if NETCOREAPP && !NETCOREAPP3_0
-        private const System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes MembersAccessedOnRead = System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicProperties;
-        private const System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes MembersAccessedOnWrite = System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicProperties;
+        private const System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes MembersAccessedOnRead = System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicProperties | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicFields;
```

Comment:
Should this rather use the autogenerated format? (ie what you get when running `/t:GenerateReferenceSource`)

Reply:
It probably should. I ran `/t:GenerateReferenceSource` on the ref project.